person.rb

```diff
@@ -25,4 +26,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(book, date)
+    Rental.new(date, self, book)
+  end
```

Comment:
- [ ] Please make sure to pass the parameters in the right order to the `Rental`'s constructor. You wrote the  `Rental`'s constructor to take the following parameters in this order:
```
def initialize(date, book, person)
```